plugins/search-backend-node/src/IndexBuilder.ts

```diff
@@ -100,28 +100,31 @@ export class IndexBuilder {
   async build() {
     return Promise.all(
       Object.keys(this.collators).map(async type => {
-        const decorators: DocumentDecorator[] = (
-          this.decorators['*'] || []
-        ).concat(this.decorators[type] || []);
+        setInterval(async () => {
+          const decorators: DocumentDecorator[] = (
+            this.decorators['*'] || []
+          ).concat(this.decorators[type] || []);
 
-        this.logger.info(
-          `Collating documents for ${type} via ${this.collators[type].collate.constructor.name}`,
-        );
-        let documents = await this.collators[type].collate.execute();
-        for (let i = 0; i < decorators.length; i++) {
           this.logger.info(
-            `Decorating ${type} documents via ${decorators[i].constructor.name}`,
+            `Collating documents for ${type} via ${this.collators[type].collate.constructor.name}`,
           );
-          documents = await decorators[i].execute(documents);
-        }
+          let documents = await this.collators[type].collate.execute();
+          for (let i = 0; i < decorators.length; i++) {
+            this.logger.info(
+              `Decorating ${type} documents via ${decorators[i].constructor.name}`,
+            );
```

Comment:
Would this be more appropriate for `debug`?  In standard practice, if the search refresh happens say every 15 minutes, seeing this for every decorator, every 15 mins, in my logs might be a lot (and repetitive) unless I wanted that level of info to debug and so then turning on `LOG_LEVEL=debug`?  just a thought!

Reply:
yesss definitely good points!